.github/workflows/aks.yaml

```diff
@@ -1,6 +1,5 @@
 # This workflow calls the main workflow with custom variables
 name: AKS-E2E
-run-name: AKS ${{ inputs.downstream_k8s_version }} on Rancher v${{ inputs.rancher_version }} deployed on ${{ inputs.k3s_version}}
```

Comment:
This does not seem to be working on scheduled jobs, so added Job summary instead.
![image](https://github.com/rancher/hosted-providers-e2e/assets/73099870/bb371e81-f0de-40cb-b7c5-0fb3c95b7276)

Reply:
It was added mainly for manually triggered jobs. The summary is nice but the point of introducing `runs-on` was to be able to identify tests without clicking on a run.

Maybe you can do something like this so that scheduled jobs are not empty:
```yaml
run-name: AKS ${{ inputs.downstream_k8s_version || '1.27.3-gke.100}} on Rancher v${{ inputs.rancher_version || '2.8-head' }} deployed on ${{ inputs.k3s_version || 'v1.27.9+k3s1' }}
```

Reply:
Done